### ISSUE TITLE
Add proper support for all oidc_logout parameters

### ISSUE THREAD

**Issue by kopernic-pl** [auth0/terraform-provider-auth0#952]
### Checklist

- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

Auth0 oidc backchannel logout configuration can be parameterized in more precise way than this provider allows.

The Auth0 Management API allows to set them as a following structure
```  "oidc_logout": {
    "backchannel_logout_urls": [
      "string"
    ],
    "backchannel_logout_initiators": {
      "mode": "custom",
      "selected_initiators": [
        "rp-logout"
      ]
    }
  },```
  
go auth0 SDK has the support for it, added https://github.com/auth0/go-auth0/pull/384 

### Describe the ideal solution

https://github.com/auth0/go-auth0/releases/tag/v1.5.0 freshest release of go sdk contains the support for setting the oidc logout parameters properly. Please adapt it 

### Alternatives and current workarounds

_No response_

### Additional context

_No response_

**Comment by kopernic-pl** [comment]
Any progress? Those are just a few boolean flags to be added...

**Comment by kopernic-pl** [comment]
I'm not sure if all initiators are supported, but this seems to be fixed by #1045 in [1.8.0](https://github.com/auth0/terraform-provider-auth0/releases/tag/v1.8.0)

**Comment by duedares-rvj** [comment]
@kopernic-pl  I tested with the first 4 among the supported initiators
rp-logout
idp-logout
password-changed
session-expired
`account-deleted`
`email-identifier-changed`

They work fine!
